docs/wiki/introduction/using-osqueryi.md

```diff
@@ -2,7 +2,7 @@
 
 ## Executing SQL queries
 
```

Comment:
As long as you're in this file, I'd add a note about how `osqueryd -S` is osqueryi